model.js

```diff
@@ -3,97 +3,135 @@ const { db } = require("./db/connection");
 const fetchAllTopics = () => {
   const sqlQuery = `SELECT * FROM topics`;
 
-  return db
-    .query(sqlQuery)
-    .then((results) => {
-      return results.rows;
-    })
-
+  return db.query(sqlQuery).then((results) => {
+    return results.rows;
+  });
 };
 
-const fetchAllArticles = () => {
-  const sqlQuery = `SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.article_id) AS comment_count
+const fetchAllArticles = (sort_by = `created_at`, order = `DESC`, topic) => {
+  const acceptedOrders = ["ASC", "DESC"];
+  const queryValues = [];
+  const acceptedTopics = ["cats", "mitch", "paper"];
```

Comment:
I don't think you need to have accepted topics - technically this can increment infinitely in the future.